### PR TITLE
Use lexis/render/Stream event to Deflect stream available via HTTP

### PR DESCRIPTION
**Pull request by tribal-tec**

### CODE REVIEW

File: plugins/extensions/plugins/DeflectPlugin.cpp [hunk 1]
```diff
@@ -187,12 +206,9 @@ void DeflectPlugin::_send(
...
     deflect::ImageWrapper deflectImage( imageData, windowSize.x(), windowSize.y(), deflect::RGBA );
 
-    ApplicationParameters& applicationParameters =
-        _brayns.getParametersManager().getApplicationParameters();
-
-    deflectImage.compressionQuality = applicationParameters.getJpegCompression();
+    deflectImage.compressionQuality = _params.getQuality();
     deflectImage.compressionPolicy =
-        ( deflectImage.compressionQuality != 100 ) ?
+        _params.getCompression() ?
```
Comment (favreau) [inline plugins/extensions/plugins/DeflectPlugin.cpp]: Following our discussion, it's probably better to have 2 different parameters for the quality and the activation of the compression. For the records, compressing to 100% gives the same quality as compression disabled, but removes the Alpha information.

**Comment by favreau** [comment]
Nice functionality! Fix the GitExternal.cmake SHA1 (51a2753 -> a853636) and it should be all good.

File: plugins/extensions/plugins/DeflectPlugin.cpp (hunk 1 above)
Comment (tribal-tec) [inline plugins/extensions/plugins/DeflectPlugin.cpp]: That's what I have now?! getCompression() is ON/OFF and there is getQuality().

**Comment by tribal-tec** [comment]
retest this please

**Comment by tribal-tec** [comment]
Needs approval for merge

**Pull request merged**